wordle-web/components/SettingsDialog.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-dialog v-model="modelValue">
+    <v-card class="ma-auto pa-4" min-width="300">

```

Comment:
I believe it's this pa-4 making it transparent

Reply:
Thank you!